docs/reference/2-ods-api/whats-new/whats-new-in-prev-v7x-releases.md

```diff
@@ -1,3 +1,7 @@
+---
+description: Summary of what changed in prior releases of 7.x
+---
```

Comment:
Adding a description like this improves the display of the DocCardList in `readme.md` - in other words, the display of child pages.

![image](https://github.com/user-attachments/assets/1b1e4542-5793-4ebc-a9a9-759dfa73b49b)